package.json

```diff
@@ -61,7 +62,10 @@
     "vue-template-compiler": "^2.6.11"
   },
   "dependencies": {
-    "launchdarkly-js-client-sdk": "^2.18.0",
     "lodash": "^4.17.19"
-  }
+  },
+  "peerDependencies": {
```

Comment:
Was worried when I saw you deleted the dependency for launchdarkly 😂 This makes sense though.